components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -2544,27 +2555,41 @@ public static void addDefinedAllSequencesToRegistry(UserRegistry registry,
 					"Error while saving defined sequences to the registry ", e);
 		} catch (IOException e) {
 			throw new APIManagementException("Error while reading defined sequence ", e);
+		} finally {
+			if (inSeqStream != null) {
+				try {
+					inSeqStream.close();
+				} catch (IOException e) {
+					log.error(
+							"Error while closing input stream in path " + seqFolderLocation + " " +
```

Comment:
@jaadds Thanks for the info and fixed in fed62f0e23f6d4ce909847a453934ee7243f51ed